### DISCUSSION TITLE
Jobs stuck and/or failing

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#3615] [Bugs]
Are there still  (or again) some server issues? Project file processing and a Packaging Jobs seem to get stuck on pending at the moment. I have one of each in a project that have been pending for roughly 3 hours now and also a delta apply job that started, but failed downloading due to timeout. Same seems to happen in a test project I use for troubleshooting.

**Comment by kiri03** [comment]
same here. I have the same problem

**Reply by Forstrevier** [reply]
Same here :-/

**Reply by colturk** [reply]
Ditto

**Comment by JUAN1936** [comment]
Idem

**Comment by suricactus** [comment]
Need to process ~500 jobs from today. There was only single worker, the issue was detected and prevented for the future. Once ready, all should function as expected.

Hope no more 3rd-party DNS issues, not the best week in QFieldCloud history.

**Reply by JUAN1936** [reply]
Thanks for all the effort. In the end everything will be fine for sure

**Reply by suricactus** [reply]
Ok, all should be working as expected.

**Reply by chrysenteron** [reply]
It's all working now. Thanks for your efforts!

**Comment by kiri03** [comment]
Everything works fine now.
Thanks
